Sources/Vapor/Multipart/MultipartParser.swift

```diff
@@ -0,0 +1,195 @@
+import CMultipartParser
+
+/// Parses multipart-encoded `Data` into `MultipartPart`s. Multipart encoding is a widely-used format for encoding/// web-form data that includes rich content like files. It allows for arbitrary data to be encoded
```

Comment:
I think you missed a newline here.
```suggestion
/// Parses multipart-encoded `Data` into `MultipartPart`s. Multipart encoding is a widely-used format for encoding
/// web-form data that includes rich content like files. It allows for arbitrary data to be encoded
```